streampark-flink/streampark-flink-shims/streampark-flink-shims-base/pom.xml

```diff
@@ -105,6 +105,19 @@
             <artifactId>hadoop-client-runtime</artifactId>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-yarn${scala.binary.flink.version}</artifactId>
+            <version>${flink.version}</version>
+            <scope>provided</scope>
+            <exclusions>
```

Comment:
Maven scope is "provided". We don't need to specify exclusions. These dependencies will be automatically excluded.

Reply:
OK, I understand this. Thank you. And it is fixed.